content/en/docs/tasks/configure-pod-container/configure-gmsa.md

```diff
@@ -253,5 +253,32 @@ If the above command corrects the error, you can automate the step by adding the
 
 If you add the `lifecycle` section show above to your Pod spec, the Pod will execute the commands listed to restart the `netlogon` service until the `nltest.exe /query` command exits without error.
 
-## GMSA limitations
-When using the [ContainerD runtime for Windows](/docs/setup/production-environment/windows/intro-windows-in-kubernetes/#cri-containerd) accessing restricted network shares via the GMSA domain identity fails. The container will receive the identity of and calls from `nltest.exe /query` will work.  It is recommended to use the [Docker EE runtime](/docs/setup/production-environment/windows/intro-windows-in-kubernetes/#docker-ee) if access to network shares is required.  The Windows Server team is working on resolving the issue in the Windows Kernel and will release a patch to resolve this issue in the future. Look for updates on the [Microsoft Windows Containers issue tracker](https://github.com/microsoft/Windows-Containers/issues/44).
+## Containerd 
+
+On Windows Server 2019 in order to use SMB Shares with Containerd, you must be running OS Build 17763.1817 (or later) which can be install using the patch [KB5000822](https://support.microsoft.com/en-us/topic/march-9-2021-kb5000822-os-build-17763-1817-2eb6197f-e3b1-4f42-ab51-84345e063564). 
+
+There is also a known issue with containerd when trying to connect to SMB shares from Pods. In some cases, once you have configured GMSA, the pod will be unable to connect to the share using the Hostname or FQDN but connecting via IP address works as expected. 
+
+```shell
+Windows PowerShell
+Copyright (C) Microsoft Corporation. All rights reserved.
+PS C:\> ping adserver.ad.local
+Pinging adserver.ad.local [192.168.111.18] with 32 bytes of data:
+Reply from 192.168.111.18: bytes=32 time=6ms TTL=124
+Reply from 192.168.111.18: bytes=32 time=5ms TTL=124
+...
+PS C:\> cd \\adserver.ad.local\test
+cd : Cannot find path '\\adserver.ad.local\test' because it does not exist.
+At line:1 char:1
++ cd \\adserver.ad.local\test
++ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    + CategoryInfo          : ObjectNotFound: (\\adserver.ad.local\test:String) [Set-Location], ItemNotFoundException
+    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.SetLocationCommand
+PS C:\> cd \\192.168.111.18\test
+PS Microsoft.PowerShell.Core\FileSystem::\\192.168.111.18\test>
+```
+
+If you encounter a case like above you should run the following `reg add "HKLM\SYSTEM\CurrentControlSet\Services\hns\State" /v EnableCompartmentNamespace /t REG_DWORD /d 1` and restart the pods.
```

Comment:
I feel like it might make more sense to just state that if you want to access smb shares by fqdn you'll need to turn this on, otherwise the above will happen and not vice versa. It's a pain that we need to turn this on but it was a precaution to not break accidentally break anything :/.

Reply:
I worded that rather oddly. Basically the "known issue" is the one we fixed, and the fix is just gated behind a registry key. So instead of "this won't work" -> "to fix turn on reg key", might make more sense to say "to enable smb access by hostname turn on reg key" -> "otherwise this won't work".

Reply:
Yeah fair point, when I wrote this last night I was thinking there would be cases when this wasn't needed but on reflection, you're right, this needs to be turned on!